vignettes/03-get-images-python.Rmd

```diff
@@ -1 +1,100 @@
-# Images Vignette
\ No newline at end of file
+---
+title: "Get Source Image Files"
+output: html_document
+---
+
+# Objective: To be able to demonstrate how to locate and retrieve RGB image files
+
+This vignette shows how to locate and retrieve image files associated with growing Season 6 
+from the University of Arizona's [Maricopa Agricultural Center](http://cals-mac.arizona.edu/) 
+using Python. The files are stored online on the data management system Clowder, 
+which is accessed using an API. We will be working with the image files generated during the 
+month of May by limiting the requests to that time period.
+
+After completing this vignette it should be possible to search for and retrieve other
+files through the use of the API.
+
+As an added bonus we've also included an exmple of how to retrieve the list of available 
+sensor names through the API. By using the sensor names returned, it's possible to retrieve
+other files containing the data the sensors have collected.
+
+## Locating the images
+
+To begin looking for files, a sensor name and site name are needed. We will be using 
+'RGB GeoTIFFs Datasets' as the sensor name and '' as the site name. Later in this
+vignette we show how to retrieve the list of available sensors.
+
+As mentioned in the overview, the url string will point to the API to use. In this case
+we'll be using "https://terraref.ncsa.illinois.edu/clowder/api" and the key will be the
+one you received in an email.
```

Comment:
... will users have received this in the email? Do we need to tell them to sign up? @max-zilla do you have suggestions for how to access the data that we are going to make public?